src/System.Runtime.Analyzers/UnitTests/DisposableTypesShouldDeclareFinalizerTests.cs

```diff
@@ -1,23 +1,557 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Test.Utilities;
 using Microsoft.CodeAnalysis.UnitTests;
-using Roslyn.Test.Utilities;
 using Xunit;
 
 namespace System.Runtime.Analyzers.UnitTests
 {
     public class DisposableTypesShouldDeclareFinalizerTests : DiagnosticAnalyzerTestBase
     {
+        [Fact]
```

Comment:
I'm going to remove these first two tests. There's another test below which shows that if you assign to a field whose type is _not_ `IntPtr`, `UIntPtr`, or `HandleRef`, then you don't get a diagnostic.